x-pack/agent/.gitignore

```diff
@@ -1,6 +1,9 @@
 # agent
 build/
 agent.dev.yml
+cmd/agent/agent
+cmd/agent/agent.yml
+cmd/agent/fleet.yml
```

Comment:
Why theses files are created?

Reply:
these are created when you build and run agent there

Reply:
i'm usually building latest in x-pack/agent/cmd/agent and enrolling to fleet. this produces some garbage i don't want to check in

Reply:
I usually use `mage build:all` at the root :)

Reply:
Maybe its would be better to have them defined in your local ignore?